SPECS/xapi.spec

```diff
@@ -95,6 +95,9 @@ Patch1010: 0010-CA-408841-rrd-don-t-update-rrds-when-ds_update-is-ca.patch
 # Fix SR.scan not atomic - cherry-picked from v25.16.0
 Patch1011: 0011-Check-that-there-are-no-changes-during-SR.scan.patch
 
+# Under review upstream
+Patch1012: 0010-xapi_guest_agent-Update-xenstore-keys-for-Windows-PV.patch
```

Comment:
strange, I expected to see 0012-xapi_guest... as we already have 11 patches applies. And if I'm not mistaken it is not under review anymore ;).